src/Mint/Common.hs

```diff
@@ -342,10 +326,12 @@ pRemove common vrange discConfig outs sigs = plam $ \pkToRemove node -> P.do
                         # out
                         #== configF.penaltyAddress
                         #&& ownInputFee
-                        #<= plovelaceValueOf
+                        #<= pvalueOf
                         # (pfield @"value" # out)
+                        # configF.stakeCS
+                        # configF.stakeTN
                   )
-                # outs -- must pay 25% fee
+                # outs -- must pay 25% penalty fee
             )
         )
```

Comment:
Shouldn't we allow withdrawing without penalty after `endStaking`?

Reply:
Yes we do allow that. If `endStaking` has passed we invoke `pClaim` instead of `pRemove` 
https://github.com/Anastasia-Labs/single-asset-staking/blob/develop/src/Mint/Standard.hs#L69